Makefile

```diff
@@ -144,15 +144,17 @@ cosmovisor:
 
 .PHONY: build build-linux build-simd-all build-simd-linux cosmovisor
 
+mockgen_cmd=go run github.com/golang/mock/mockgen
```

Comment:
It seems to me that you're treating this as if it was a bash variable. Makefile vars and bash vars behave differently, thus I'd advise treading carefully. Furthermore, may I ask what's the benefit of replacing `mockgen` with `go run [...]`?

Reply:
Thanks for note. What's wrong in the assignment above? I was always using it and it works. Here is a [documentation](https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_6.html#SEC64) about setting a variable, and it looks to be compatible with what I'm using above.

Reply:
> Furthermore, may I ask what's the benefit of replacing mockgen with go run

The goal is to use the `mockgen` from the module dependencies, rather than "global" installation.